src/lib/rnp.cpp

```diff
@@ -6770,6 +6780,26 @@ try {
 }
 FFI_GUARD
 
+rnp_result_t
```

Comment:
Please use `RNP_EXPERIMENTAL_PQC` here like it was done for `rnp_op_generate_set_sphincsplus_param`

Reply:
I meant to put the whole function under the define, so if RNP_EXPERIMENTAL_PQC is not defined then it absent from compiled code and not available in the header.